libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigator.java

```diff
@@ -13,7 +13,8 @@
 
 import java.util.Date;
 
-class MapboxNavigator {
+// TODO internal
+public class MapboxNavigator {
```

Comment:
final? Also remove 'TODO'

Reply:
Nah, that's a `TODO` that we need to check before merging `libandroid-navigation-internal` into `master`: Could `MapboxNavigator` live in the `internal` package?